uiowa/uiowa
# UIPress: Added logic to teaser to display editor if author is empty

Resolves https://github.com/uiowa/uiowa/issues/5578

# How to test

- `ddev blt ds --site=uipress.uiowa.edu`
- Code review
- Look at Fame and Fandom book on the homepage: https://uipress.uiowa.edu/ (refresh)
- Verify that editor(s) display if author is missing
